spec/suma/member/dashboard_spec.rb

```diff
@@ -18,4 +18,18 @@
       program_enrollments: have_same_ids_as(pe1, pe2),
     )
   end
+
+  it "sorts enrollments by program ordinal" do
+    program_fac = Suma::Fixtures.program
+    p3 = program_fac.create(ordinal: 3)
+    p2 = program_fac.create(ordinal: 2)
```

Comment:
Always vary the order so it's not ascending/descending- make p3, then p1, then p2, something like that (or p1, p3, p2, whatever, just never p1/2/3, or p3/2/1)